src/graphql/queries/ListArticles.js

```diff
@@ -402,6 +409,21 @@ export default {
       });
     }
 
+    if (filter.includeHighlight) {
+      body.highlight = {
+        order: 'score',
+        fields: {
+          text: {
```

Comment:
From [Figma](https://www.figma.com/file/zpD45j8nqDB2XfA6m2QskO/Cofacts-website?node-id=585%3A168) we will need to highlight hyperlink title & hyperlink descriptions as well:

![image](https://user-images.githubusercontent.com/108608/90856001-83510e80-e3b3-11ea-83b6-690527ba1601.png)

![image](https://user-images.githubusercontent.com/108608/90855697-cf4f8380-e3b2-11ea-934f-a0ae8001b240.png)

- We will need highlight on `text`, `hyperlinks.title` and `hyperlinks.summary`
- `highlight` output may be more complex than a simple string. We may need a structure like `[{field: 'text', highlight: 'OOOOOOO'}]` to tell UI to highlight specific parts.

Reply:
I have some problem in highlighting hyperlink, details updated in the PR description.